config/foreman-proxy-content.migrations/210415180240-enable-katello-agent.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman_proxy_content'].is_a?(Hash) && answers['foreman_proxy_content'].key?('qpid_router')
+  answers['foreman_proxy_content']['enable_katello_agent'] = answers['foreman_proxy_content']['qpid_router']
```

Comment:
This may be a bit more clear that it's really moving it:

```suggestion
  answers['foreman_proxy_content']['enable_katello_agent'] = answers['foreman_proxy_content'].delete('qpid_router')
```